README.md

```diff
@@ -138,55 +135,50 @@ For reference here's how a full configuration could look:
+| `files`                            | _optional_  | Glob patterns to detect Synthetic test [configuration files][2].                                                                                                                                                                 |
 | `jUnitReport`                      | _optional_  | The filename for a JUnit report if you want to generate one.                                                                                                                                                                        |
-| `mobileApplicationVersion`         | _optional_  | Override the default mobile application version for a Synthetic mobile application test. The version must be uploaded and available within Datadog. This version is also outputted by the [`datadog-mobile-app-upload` step][1002]. |
-| `mobileApplicationVersionFilePath` | _optional_  | Override the application version for Synthetic mobile application tests.                                                                                                                                                            |
+| `mobileApplicationVersion`         | _optional_  | Override the default mobile application version for a Synthetic mobile application test. The version must be uploaded and available within Datadog. This version is also outputted by the [`datadog-mobile-app-upload` step][11]. |
+| `mobileApplicationVersionFilePath` | _optional_  | Override the application version for [Synthetic mobile application tests][19].                                                                                                                                                            |
 | `pollingTimeout`                   | _optional_  | The duration (in milliseconds) after which the batch is deemed as failed because of a timeout. The default is 30 minutes.                                                                                                           |
 | `publicIds`                        | _optional_  | String of public IDs separated by commas for Synthetic tests you want to trigger.                                                                                                                                                   |
-| `site`                             | _optional_  | The Datadog site to send data to. If the `DD_SITE` environment variable is set, it takes precedence.                                                                                                                                |
+| `site`                             | _optional_  | The [Datadog site][16] to send data to. If the `DD_SITE` environment variable is set, it takes precedence. Your default site is {{< region-param key="dd_site" code="true" >}}.                                                                                                                               |
```

Comment:
Could we avoid using `{{ }}` markup in publicly available markdown documents? It'll be displayed as is. Instead I believe we should use `<!-- -->` markup, like [on the datadog-ci synthetics readme](https://github.com/DataDog/datadog-ci/blob/bff12e212215133d0eb59ae1814d052a2ee886c2/src/commands/synthetics/README.md?plain=1#L68).
```suggestion
| `site`                             | _optional_  | The [Datadog site][16] to send data to. If the `DD_SITE` environment variable is set, it takes precedence. <!-- partial Your default site is {{< region-param key="dd_site" code="true" >}}. partial -->.                                                                                                                               |
```